model-viewer/src/main/java/net/runelite/modelviewer/ModelViewer.java

```diff
@@ -843,6 +909,21 @@ public static int RS2HSB_to_RGB(int RS2HSB)
 		int decode_hue = (RS2HSB >> 10) & 0x3f;
 		int decode_saturation = (RS2HSB >> 7) & 0x07;
 		int decode_brightness = (RS2HSB & 0x7f);
-		return Color.HSBtoRGB((float) decode_hue / 63, (float) decode_saturation / 7, (float) decode_brightness / 127);
+		int rgb = Color.HSBtoRGB((float) decode_hue / 63, (float) decode_saturation / 7, (float) decode_brightness / 127);
+		return adjustRGB(rgb & 0xFFFFFF, 0.6);
+	}
+
+	private static int adjustRGB(int rgb, double brightness)
+	{
+		double rp = (double) (rgb >> 16) / 256.0D;
+		double gp = (double) (rgb >> 8 & 255) / 256.0D;
+		double bp = (double) (rgb & 255) / 256.0D;
+		rp = Math.pow(rp, brightness);
```

Comment:
This noticeably destroys performance. Maybe use precomputed colors like the client.

Reply:
Added a new method to use in place of this one. I have kept it for reference though.